compose.yaml

```diff
@@ -1,16 +1,19 @@
 services:
-  mysqldb:
-   image: mysql
-   command: --default-authentication-plugin=mysql_native_password
+  mariadb:
+   image: mariadb:10.3 # the version installed on prod
```

Comment:
Rename to be clearer and specify the same version running on our VMs. 

We can remove this when we're on PG in all envs.